### PR TITLE
Add new token prices for Scroll

### PR DESCRIPTION
**Pull request by Jam516**
Add new token prices for Scroll

### CODE REVIEW

**Comment by Jam516** [comment]
Hi @jeff-dude , how do I deal with this error
Screenshot 2024-06-28 110651

First time adding to prices.usd so I'm not familiar with it

File: models/prices/scroll/prices_scroll_tokens.sql [hunk 1]
```diff
@@ -22,6 +22,14 @@ FROM
     ('izi-izumi-finance', 'scroll', 'iZi', 0x60D01EC2D5E98Ac51C8B4cF84DfCCE98D527c747, 18),
     ('lusd-liquity-usd', 'scroll', 'LUSD', 0xeDEAbc3A1e7D21fE835FFA6f83a710c70BB1a051, 18),
     ('wsteth-wrapped-liquid-staked-ether-20', 'scroll', 'wstETH', 0xf610A9dfB7C89644979b4A0f27063E9e7d7Cda32, 18),
-    ('reth-rocket-pool-eth', 'scroll', 'rETH', 0x53878B874283351D26d206FA512aEcE1Bef6C0dD, 18)
-
+    ('reth-rocket-pool-eth', 'scroll', 'rETH', 0x53878B874283351D26d206FA512aEcE1Bef6C0dD, 18),
+    ('dai-dai', 'scroll', 'DAI', 0xca77eb3fefe3725dc33bccb54edefc3d9f764f97, 18),
+    ('stone-stakestone-ether', 'scroll', 'STONE', 0x80137510979822322193fc997d400d5a6c747bf7, 18),
+    ('itp-interport-token', 'scroll', 'ITP', 0x2b1d36f5b61addaf7da7ebbd11b35fd8cfb0de31, 18),
```
Comment (Hosuke) [inline models/prices/scroll/prices_scroll_tokens.sql:28]: Not sure if this token is still active for now, should we remove it?
https://api.coinpaprika.com/v1/coins/itp-interport-token

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: models/prices/scroll/prices_scroll_tokens.sql (hunk 1 above)
Comment (Jam516) [reply models/prices/scroll/prices_scroll_tokens.sql:28]: we can remove it, its not a major one. will make a commit for that now

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @Jam516

**Pull request merged**